addons/account/account_move_line.py

```diff
@@ -1359,7 +1359,7 @@ def create(self, cr, uid, vals, context=None, check=True):
             account_id = 'account_collected_id'
             base_sign = 'base_sign'
             tax_sign = 'tax_sign'
-            if journal.type in ('purchase_refund', 'sale_refund') or (journal.type in ('cash', 'bank') and total < 0):
+            if journal.type in ('purchase_refund', 'sale_refund') or (journal.type in ('cash', 'bank') and total < 0 and tax_id.type_tax_use != 'sale'):
```

Comment:
hello,

can you explain the reason of this modification please? It seems to me that we could accept the patch without that line and it would be just good enough...

thanks,